### ISSUE TITLE
[a11y-defect-2]: Color contrast - First-party evidence request red "past due" text must be at least 4.5:1 contrast ratio on white backgroundas

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#109485]
### Point of contact

@1copenut

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

In our first-party evidence requests we're using a light red text for overdue request notices. This text does not meet the WCAG 2 required 4.5:1 contrast ratio with a white background. Screenshots attached.

---

![Image](https://github.com/user-attachments/assets/88c73d43-b9e0-40d6-b115-32f46860ef94)

---

![Image](https://github.com/user-attachments/assets/4153f089-eb83-413e-ba2e-0d94b33e46f3)

### Reproduction steps

1. Log into staging, using a test user who has new first and third-party evidence requests. I tested with 211.
2. Go to `/track-claims/your-claims` after logging in
3. Click on a claim that has multiple first party claims and look for one that has a due date prior to today's date
4. Click on that claim "Details" link
5. Open dev tool and verify the text contrast ratio is too low, or compare it to the `#fff` white background on a site like [Who can use](https://whocanuse.com)

### Proposed solution or next steps

Will need a design okay from @sh0wn but I'm guessing the darker red `vads-color-red-dark` ( #b50909 ) will be the right call.

### References, articles, or WCAG support

1. [Understanding SC 1.4.3: Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Voice command
- [ ] Zoom
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by sh0wn** [comment]
@janechodance Looks good. Compared the original color to the recommended red, which is `vads-color-error-dark` in the Figma design system.

![Image](https://github.com/user-attachments/assets/cca7d3ec-83c2-4dd4-a65e-3b7116d51246)

**Comment by 1Copenut** [comment]
@salihaghaffar This one's good to move to done.

---

![Image](https://github.com/user-attachments/assets/7a656176-b6be-46d0-9fec-0c2edf64c974)
